src/mmw/apps/modeling/calcs.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.contrib.gis.geos import GEOSGeometry
+
+from apps.modeling.mapshed.calcs import animal_energy_units
+
+ANIMAL_DISPLAY_NAMES = {
+    'sheep': 'Sheep',
+    'horses': 'Horses',
+    'turkeys': 'Turkeys',
+    'layers': 'Layers',
+    'beef_cows': 'Beef Cows',
+    'hogs': 'Hogs',
+    'dairy_cows': 'Dairy Cows',
+    'broilers': 'Broilers',
+}
+
+
+def animal_population(geojson):
+    """
+    Given a GeoJSON shape, call MapShed's `animal_energy_units` method
+    to calculate the area-weighted county animal population. Returns a
+    dictionary to append to the outgoing JSON for analysis results.
+    """
+    geom = GEOSGeometry(geojson, srid=4326)
+    aeu_for_geom = animal_energy_units(geom)[2]
+    aeu_return_values = []
+
+    for animal, aeu_value in aeu_for_geom.iteritems():
+        animal_dictionary = {
+            'type': ANIMAL_DISPLAY_NAMES[animal],
+            'aeu': aeu_value,
+        }
+        aeu_return_values.append(animal_dictionary)
+
+    return {
+        u'displayName': 'Animals',
+        u'name': 'livestock',
```

Comment:
Consider renaming `livestock` to `animals` here too, so that they name and display name stay in sync and are easier to reason about. Best do this now before it has a chance to permeate the system.